server/channels/app/plugin_install.go

```diff
@@ -174,6 +174,16 @@ func (ch *Channels) installPlugin(pluginFile, signature io.ReadSeeker, installat
 		mlog.Warn("Failed to notify plugin status changed", mlog.Err(err))
 	}
 
+	ch.cfgSvc.UpdateConfig(func(cfg *model.Config) {
+		if _, ok := cfg.PluginSettings.Plugins[manifest.Id]; !ok {
+			cfg.PluginSettings.Plugins[manifest.Id] = make(map[string]any)
+		}
+
+		for _, pluginSetting := range manifest.SettingsSchema.Settings {
+			cfg.PluginSettings.Plugins[manifest.Id][pluginSetting.Key] = pluginSetting.Default
```

Comment:
I'm not completely sure but I feel like keys are always lowercase at pluginsettings.

Look for examples at [1](https://github.com/mattermost/mattermost-server/blob/master/server/channels/app/plugin_api.go#L47) and [2](https://github.com/mattermost/mattermost-server/blob/master/server/channels/app/plugin_api.go#L53).

```suggestion
			cfg.PluginSettings.Plugins[manifest.Id][strings.ToLower(pluginSetting.Key)] = pluginSetting.Default
```

Reply:
@trilopin I'm not sure if the plugin settings are always saved this way. Here's an example from my local config:

![CleanShot 2023-05-02 at 15 59 58](https://user-images.githubusercontent.com/6913320/235772852-e4f4f4f4-b8c7-4826-b02c-2aa3632f4e89.png)

What are the benefits/risks for putting this `ToLower` here? I think the Golang JSON unmarshaler takes care of this, but I want to err on the side of caution here.